python/lsst/meas/algorithms/detection.py

```diff
@@ -166,154 +234,154 @@ def setDefaults(self):
 
 
 class SourceDetectionTask(pipeBase.Task):
-    """!
-@anchor SourceDetectionTask_
-
-@brief Detect positive and negative sources on an exposure and return a new @link table.SourceCatalog@endlink.
-
-@section meas_algorithms_detection_Contents Contents
+    """Detect positive and negative sources on an exposure and return a new @link table.SourceCatalog
```

Comment:
Okay, documenting `SourceDetectionTask` is a bit tricky because we're in the middle of redesigning how task documentation works (this is my current [DM-14522](
https://jira.lsstcorp.org/browse/DM-14522)) epic. The gist of it is that a lot of what's currently in the docstrings of Task subclasses will actually get moved out of docstrings and into `.rst` pages for each Task.

I think you can still do enough of a conversion that these docstrings are numpydoc-able.